pydicer/convert/HN_137_RTstruct_to_Niffti_all - PET.py

```diff
@@ -0,0 +1,109 @@
+###################### Read Me #######################
+# This file is to convert .dicom files into .nifti files for PET imaging data, which can work on all patients.
+# Below is an example about how to use it.
+# BTW, from pet_to_nifti_convert_new import convert_dicom_to_nifty is an important function for converting PET's nifti files.
+#############Coded by Shuchao Pang 13 NOV 2020########
+
+import sys
+import os
+sys.path.append('../../..')
+import SimpleITK as sitk
+import pydicom
+import glob

```

Comment:
In pydicer we use Pathlib's Path class to handle all paths... This provides glob and much more.